interfaces/IProvisioning.h

```diff
@@ -35,6 +35,8 @@ namespace Exchange {
 
         virtual void Register(IProvisioning::INotification* provisioning) = 0;
         virtual void Unregister(IProvisioning::INotification* provisioning) = 0;
+        virtual uint32_t DRMId (const string& label /* @in */, string& drmId /* @out */) const = 0;
```

Comment:
The DRMId is binary data, so let please convey this using a length field in and a uint8_t buffer[] field out.